Add lodash.intersection as a dependency

My guess is a supplementary package required lodash and has since dropped it (or specified specifics and it did not include intersection). Lodash is currently in the devDeps, but lodash/intersection is used in src/normalize-options.js.

The suggestion is to `npm install --save lodash.intersection`, in order to make all of lodash a dependency just for the one function.

This currently has our build process broken, though there are workarounds at the moment.

The error manifests itself as:
```
Error: Cannot find module 'lodash/intersection' (While processing preset: "<custom-preset>/lib/index.js")
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/opt/signal/node_modules/babel-preset-env/lib/normalize-options.js:7:21)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
```

--- a/src/normalize-options.js
+++ b/src/normalize-options.js
@@ -1,6 +1,4 @@
-import intersection from "lodash/intersection";
 import invariant from "invariant";
-
 import builtInsList from "../data/built-ins.json";
 import defaultInclude from "./default-includes";
 import moduleTransformations from "./module-transformations";
@@ -38,7 +36,9 @@ export const validateIncludesAndExcludes = (opts = [], type) => {
 };
 
 export const checkDuplicateIncludeExcludes = (include = [], exclude = []) => {
-  const duplicates = intersection(include, exclude);
+  const duplicates = include.filter(
+    (opt) => exclude.indexOf(opt) >= 0
+  );
 
   invariant(
     duplicates.length === 0,

